### DISCUSSION TITLE
Navigation issue after upgrading to Nova 4

### DISCUSSION THREAD

**Discussion by ObaydaAlesawi** [laravel/nova-issues#4352] [Q&A]
- Laravel Version: 9.12
- Nova Version: 4.0
- PHP Version: 8.1

### Description:
After upgrading to nova 4, I'm facing an issue with all links as shown in the screenshot, links are working if you try to change it from the link bar itself, but not when clicking on it
![image](https://user-images.githubusercontent.com/32073999/171395163-f98be69c-1077-4ffb-a30a-dbf5e3f578e5.png)

**Comment by crynobone** [comment]
This is the first time we have seen such an issue and we are unable to help without seeing a more detailed description and code. 

If possible please submit reproducing repository similar to https://github.com/nova-issues in order for us to debug this issue.
